upstream/uphttp/upstream.go

```diff
@@ -46,9 +46,6 @@ func AppRootURL(app apps.App, _ string) (string, error) {
 }
 
 func (u *Upstream) Roundtrip(app apps.App, creq apps.CallRequest, async bool) (io.ReadCloser, error) {
-	if !app.Manifest.SupportsDeploy(apps.DeployHTTP) {
```

Comment:
Why is this check not longer needed? `GetStatic` still uses it.

Reply:
Heh, it was actually misfiring for OpenFaaS-only apps, that's why I removed it. The check comes exclusively from (http.)`GetRootURL`, it's the "bottleneck", the right place to check.